modules/otel-collector-components.adoc

```diff
@@ -199,6 +199,109 @@ The OpenCensus receiver provides backwards compatibility with the OpenCensus pro
 Wildcards with `+*+` are accepted under the `cors_allowed_origins`.
 To match any origin, enter only `+*+`.
 
+[id="k8scluster-receiver_{context}"]
+=== Kubernetes cluster receiver
+
+The Kubernetes cluster receiver is responsible for gathering cluster-level metrics and entity events from the Kubernetes API server. It utilizes the K8s API to stay informed about updates. Authentication for this receiver is exclusively supported through service accounts.
+
+.OpenTelemetry Collector custom resource with the enabled Kubernetes cluster receiver
+[source,yaml]
+----
+  receivers:
+    k8s_cluster:
```

Comment:
Support for collecting OpenShift specific metrics is also available which needs to be documented. https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sclusterreceiver#openshift

Reply:
Done

Reply:
@IshwarKanse, please ping me when you get around to verifying and approving Ruben's update.

Reply:
Thank you both.